docs/docs/guides/labs/dg/index.md

```diff
@@ -7,21 +7,55 @@ import Preview from '@site/docs/partials/\_Preview.md';
 
 <Preview />
 
-`dg` is a new command line interface that provides a streamlined Dagster development experience. It is a drop-in replacement for the Dagster CLI that can be [used in existing projects](/guides/labs/dg/incrementally-adopting-dg/migrating-project) or used to [scaffold new Dagster projects](/guides/labs/dg/scaffolding-a-project). Once a project is set up to use `dg`, you can list, check, and scaffold Dagster definitions and [components](/guides/labs/components/) with ease. `dg` is designed to be globally installed and has no dependency on `dagster` itself.
+`dg` is a new command line interface that provides a streamlined Dagster development experience. It is a drop-in replacement for the Dagster CLI that can be [used in existing projects](/guides/labs/dg/incrementally-adopting-dg/migrating-project) or used to [scaffold new Dagster projects](/guides/labs/dg/scaffolding-a-project). Once a project is set up to use `dg`, you can list, check, and scaffold Dagster definitions and [components](/guides/labs/components/) with ease. `dg` is designed to be usable from an isolated environment and has no dependency on `dagster` itself.
 
 ## Installation
 
-### Install `uv`
+There are two basic approaches to installing `dg`:
 
-import InstallUv from '@site/docs/partials/\_InstallUv.md';
+- A _local_ `dg` installation installs `dg` into a particular user-managed virtual
+  environment. The `dg` executable will be available when the
+  virtual environment is active. This is the traditional workflow for Python
+  developer tools. A local installation can be done with any Python package manager.
+- A _global_ `dg` installation installs `dg` into a hidden, isolated Python
+  environment. The `dg` executable is always available in the user's `$PATH`,
+  regardless of any virtual environment activation in the shell. A global
+  installation is simpler, only needs to be done once, and better supports
+  multiple Python projects. We recommend a global installation for most users.
+  A global installation requires the Python package manager [`uv`](https://docs.astral.sh/uv/).
 
-<InstallUv />
+<Tabs>
+<TabItem value="local" label="local installation">
+    Activate your virtual environment (if using one) and run:
 
-### Install `dg`
+    ```
+    pip install dagster-dg
+    ```
+</TabItem>
+<TabItem value="global" label="global installation">
 
-import InstallDg from '@site/docs/partials/\_InstallDg.md';
+First, install the Python package manager `uv`:
 
-<InstallDg />
+<CliInvocationExample contents="brew install uv" />
```

Comment:
we have not made this work yet. we don't have a brew entry.

Reply:
That's `uv`, not `dg`